onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -80,15 +81,31 @@ bool NodeGroupSelector::CheckQDQNodes(const GraphViewer& graph_viewer, const Nod
 
 std::optional<NodeGroup> NodeGroupSelector::GetQDQSelection(const GraphViewer& graph_viewer, const Node& node) const {
   std::vector<const Node*> dq_nodes = FindQDQNodes(graph_viewer, node, true);
-  std::vector<const Node*> q_nodes = FindQDQNodes(graph_viewer, node, false);
-  if (!Check(graph_viewer, node, dq_nodes, q_nodes)) {
+  const Node* p_activation_node = nullptr;
+  if (node.GetOutputEdgesCount() == 1) {
```

Comment:
This is only handling the case for nodes with one output. Could this be made more generic to handle nodes with multiple outputs (e.g., Split)?

Reply:
Split is tricky, it produces multiple outputs, each of them may have their own Clip and Q with different scale/zp. Maybe a single redundent_clip_node in the NodeUnit is not enough, may need a vector. I think we can think of this if we see such case in real model.